test/e2e/installplan_e2e_test.go

```diff
@@ -3247,13 +3226,12 @@ var _ = Describe("Install Plan", func() {
 			outIP.Status = ip.Status
 			_, err = crc.OperatorsV1alpha1().InstallPlans(generatedNamespace.GetName()).UpdateStatus(context.Background(), outIP, metav1.UpdateOptions{})
 			Expect(err).NotTo(HaveOccurred())
+
```

Comment:
This is no needed.

Reply:
Woops, yeah I forgot to push up that change.